frontend/src/lib/StaticGraphParser.ts

```diff
@@ -106,7 +114,11 @@ function buildTektonDag(graph: dagre.graphlib.Graph, template: any): void {
       task['runAfter'].forEach((depTask: any) => {
         graph.setEdge(depTask, taskName);
       });
-
+    if (anyTasks.includes(task['name'])) {
```

Comment:
Could we add a comment:
`// Adds dependencies for anySequencers from 'anyCondition' annotation`
Or something similar?